test/index.js

```diff
@@ -6,69 +6,53 @@ const Code = require('code');
 const Hapi = require('hapi');
 const Lab = require('lab');
 const Redis = require('../');
-
-// Declare internals
-
-const internals = {};
```

Comment:
`internals` [is mandatory](https://github.com/hapijs/contrib/blob/master/Style.md#module-globals)